add setup steps in tutorial (#1015)
fzyzcjy/flutter_rust_bridge: book/src/tutorial_with_flutter.md

git clone https://github.com/fzyzcjy/flutter_rust_bridge && cd flutter_rust_bridge/frb_example/with_flutter
```

## Setup dependencies

> For non-debian based distributions a few tricks required.

Reviewer: ```suggestion
For non-debian based distributions, there are a few prerequisites:
```

Ensure that packages are up to date (or install by demand).

- clang
- llvm-libs
- glibc

Set environment variable in your shell profile (`.bashrc`, `.zshrc`, etc).

```bash
export CPATH="$(clang -v 2>&1 | grep "Selected GCC installation" | rev | cut -d' ' -f1 | rev)/include"

> ```

Reviewer: So, you mentioned "non-debian" users. This includes:

- [ ] windows
- [ ] linux (except debian)
- [ ] macos (M1, Intel)

Is it true that all these kinds of users need the setup? (IMHO I myself do not need it; I am on macos intel)

Author: You are right, I mean Linux other than debian. At least during issues research I have not met others
